srml/support/src/lib.rs

```diff
@@ -134,3 +134,16 @@ pub use mashup::*;
 #[cfg(feature = "std")]
 #[doc(hidden)]
 pub use serde_derive::*;
+
+#[macro_export]
+#[doc(hidden)]
```

Comment:
I would like to have a short explanation for this macro and `doc(hidden)` removed. Otherwise the macro will be invisible in the documentation.